compiler/infra/BitVector.hpp

```diff
@@ -163,6 +173,32 @@ class TR_BitVector
    TR_BitVector() : _numChunks(0), _chunks(NULL), _firstChunkWithNonZero(0), _lastChunkWithNonZero(-1), _growable(growable), _region(0) { }
    TR_BitVector(TR::Region &region) : _numChunks(0), _chunks(NULL), _firstChunkWithNonZero(0), _lastChunkWithNonZero(-1), _growable(growable), _region(&region) { }
 
+   /**
+    * @brief Constructor to create a new BitVector by reading serialized data from the memory buffer
+    * @param [in] buffer Memory buffer containing serialized BitVector
+    */
+   TR_BitVector(uint8_t * &buffer)
```

Comment:
This needs similar documentation to `serialize` to note how much data should be read. This constructor implicitly assumes there is a certain amount of data present and it reads memory unconditionally.